kafka/admin/kafka.py

```diff
@@ -243,6 +245,44 @@ def _refresh_controller_id(self):
                 "The controller appears to be running Kafka {}. KafkaAdmin requires brokers >= 0.10.0.0."
                 .format(version))
 
+    def _find_group_coordinator_id(self, group_id):
+        """Find the broker node_id of the coordinator of the given group.
+
+        Sends a FindCoordinatorRequest message to the cluster. Will block until
+        the FindCoordinatorResponse is received. Any errors are immediately
+        raised.
+
+        :param group_id: The consumer group ID. This is typically the group
+            name as a string.
+        :return: The node_id of the broker that is the coordinator.
+        """
+        # Note: Java may change how this is implemented in KAFKA-6791.
+        #
+        # TODO add support for dynamically picking version of
+        # GroupCoordinatorRequest which was renamed to FindCoordinatorRequest.
+        # When I experimented with this, GroupCoordinatorResponse_v1 didn't
+        # match GroupCoordinatorResponse_v0 and I couldn't figure out why.
```

Comment:
I believe that v1 adds the idea of a group type. All groups I have seen are consumers, but I think Kafka Streams may use group coordination for producers or even other abstract groups. In any event, we'd need to add a parameter for group type I believe.

Reply:
Good thought.

I'm going to merge this for now and will add that as part of switching it to a FindCoordinatorRequest (which is a bigger change of renaming things across the client/consumer code).

Reply:
Emm, TransactionCoordinator is looked up by that too)